app/src/main/java/net/aiscope/gdd_app/ui/capture/CaptureImageActivity.kt

```diff
@@ -34,6 +38,30 @@ class CaptureImageActivity : AppCompatActivity(), CaptureImageView {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_capture_image)
         setSupportActionBar(toolbar)
+        toolbar.setNavigationOnClickListener {
+            val alertDialog: AlertDialog? = this.let {
+                val builder = AlertDialog.Builder(it, R.style.MyAlertDialogStyle)
+                builder.apply {
+                    setPositiveButton(R.string.ok
+                    ) { _, _ ->
+                        // User clicked OK button
+                        val intent = Intent(context,  MainActivity::class.java)
```

Comment:
I think that it's also needed to clear the activity stack here, otherwise, the user will be able to return back to this activity by using the back button, which should not be allowed. See the `goToHome()` method in the `MetadataActivity`.